dist/cjs/lib/getDisplayValue.js

```diff
@@ -7,7 +7,7 @@ function getDisplayValue(value) {
   if (value && typeof value === 'object') {
     if (Array.isArray(value)) {
       return value.map(function (singleOption) {
-        return singleOption.name;
+        return singleOption && singleOption.name;
```

Comment:
Why not just `return singleOption?.name;` ?
This JSX's `&&` confuses comparison with conditional inside a JS file to me.

Reply:
Agree. If we go with optional chaining though, make sure the babel plugin is included in the cjs (and esm otherwise breaking change) build. Or is it covered by @babel/env already?

Reply:
To be honest I did not touch this file. I just built it. Probably the version thing for babel... Strange.